tests/data/good/d.md

```diff
@@ -0,0 +1,10 @@
+Talk talk talk.
+
+```python
+1 + 1 == 2
+```
+
+This is not parsed as code.
+
+```python
+
```

Comment:
I'm thinking out loud ... technically this isn't valid markdown without the closing ticks ... I don't think this tool should raise the error, but I wonder if the test example is better with closing ticks. Got an opinion on this?

Reply:
No, it's not valid markdown. I wrote this as a whitebox test after reading the `grab_code_blocks` implementation and noting that it would ignore such typos. Probably actually out of scope to test in this project.
I could add an empty code block for testing if you would like, but it should probably go in the good data bucket, maybe `tests/data/good/c.md` as that is already ambiguous language?

Reply:
Yeah, the empty codeblock might be better I think. Putting it in `tests/data/good/c.md` sounds like a good idea 👍